client/ayon_blender/plugins/publish/extract_fbx_animation.py

```diff
@@ -132,13 +133,35 @@ def process(self, instance):
         max_frame = min(starting_frames)
         min_frame = max(ending_frames)
 
-        # We bake the copy of the current action for each object
-        bpy_extras.anim_utils.bake_action_objects(
-            object_action_pairs,
-            frames=range(int(min_frame), int(max_frame)),
-            do_object=False,
-            do_clean=False
-        )
+        ver_major, _, _ = get_blender_version()
+        if ver_major >= 4:
+            # We bake the copy of the current action for each object
+            bake_options = bpy_extras.anim_utils.BakeOptions(
+                only_selected=True,
```

Comment:
The old way also has the `only_selected` argument it seems, which defaults to `False` and wasn't used.... was that actually a bug and should the old way also specify `only_selected=True`? or should the new way also be using `False`?

It seems we're not making any selections - as such, `only_selected=True` seems wrong?

```suggestion
                only_selected=False,
```